packages/devtools_app/lib/src/shared/ui/gtags.dart

```diff
@@ -30,22 +30,24 @@ class GTag {
   static const String _event = 'event';
   static const String _exception = 'exception';
 
-  static bool get shouldSendAnalytics => kReleaseMode || _debugAnalytics;
+  static bool get _shouldSendAnalytics => kReleaseMode;
```

Comment:
get rid of `_shouldSendAnalytics` and use `kReleaseMode` directly now that it is a getter for one variable